jaraco/abode/event_controller.py

```diff
@@ -186,7 +187,7 @@ def _on_socket_connected(self):
             # Callbacks should still execute even if refresh fails (Abode
             # server issues) so that the entity availability in Home Assistant
             # is updated since we are in fact connected to the web socket.
-            for callbacks in self._connection_status_callbacks.values:
+            for callbacks in self._connection_status_callbacks.values():
```

Comment:
Looks like this was a genuine bug introduced in d0adb718b9d89a0280f4c004d15f03f51e5856a8, suggesting we really should get some test coverage for this module.